### DISCUSSION TITLE
OTA update zbbridge

### DISCUSSION THREAD

**Discussion by pikimk** [arendst/Tasmota#12047] [Q&A] [answered]
Hello, im having a problem with sonoff zbbridge, where i cannot update with OTA. The OTA works fine on esp8266 module, on esp32 module, but its not working on sonoff zbbridge (it is using esp8266 chip inside besides the cc2530) so im thinking maybe the spi flash chip on the bridge is not enough for OTA. Can anyone help me?

**Comment by barbudor** [comment, marked as answer]
There should be no reasons but without anymore details it is impossible to help
Provide more details on what you did, which url you used etc...

**Reply by pikimk** [reply]
Ok, so i was working with tasmota 9.3.0 and sonoff zbbridge, and first i had to flash tasmota-zbbridge  and then the cc2530 firmware. with 9.3.0 i was having a hard time because without the cc2530 firmware, tasmota kept restarting. so i tried 9.4.0 and 9.2.0 and both were working fine (they didnt restart without cc530 firmware), but only with 9.2.0 i was able to update via ota from the official tasmota servers. i also made local server and i was able to update via ota on 9.2.0. i think on 9.4.0 it gave me Exeption 9 on the console. So then i tried to upload the cc2530 firmware via OTA and the bridge crashed and i had to flash it with FTDI programmer. So my first goal was to install the cc2530 firmware via OTA not the tasmota Web server. Any help?

**Reply by sfromis** [reply]
With the Sonoff ZbBridge, you should certainly not flash CC2530 firmware, but the [provided](https://zigbee.blakadder.com/Sonoff_ZBBridge.html) EZSP firmware instead.

**Reply by pikimk** [reply]
Yes, by cc2530 firmware i meant the provided ezsp firmware. ncp-uart-sw_6.7.8_115200.ota
